### ISSUE TITLE
Initial Deploy Script Syntax Error

### ISSUE THREAD

**Issue by weklund** [arminc/terraform-ecs#19]
I've looked at deploy.sh, I can't seem to find the root of an error I keep getting after cloning master:

```
./deploy.sh CONTAINER_VERSION=nginx:alpine create

Error parsing parameter 'cli-input-json': Invalid JSON: Expecting value: line 6 column 16 (char 95)
JSON received: {
  "family": "nginx",
  "containerDefinitions": [
    {
      "name": "nginx",
      "image": ,
      "memory": 128,
      "portMappings": [
        {
          "containerPort": 80,
          "protocol": "tcp"
        }
      ]    
    }
  ]
}
```

**Comment by sleavitt** [comment]
The issue with that JSON is that there is no valid value for the "image" node. Whatever is generating that JSON is apparently not populating a proper value for "image".

**Comment by Patrik-Stas** [comment]
I've opened PR to fix this https://github.com/arminc/terraform-ecs/pull/31

**Comment by arminc** [comment]
Fixed by #31
